lib/ClimaLandSimulations/src/Fluxnet/fluxnet_utilities/inputs_dataframe.jl

```diff
@@ -1,11 +1,25 @@
 export make_inputs_df
 
 """
-    make_inputs_df(LOCAL_DATETIME, drivers)
+    make_inputs_df(site_ID)
 
 Returns DataFrames of inputs, one unitless, one in SI units, one in commonly used units. 
 """
-function make_inputs_df(LOCAL_DATETIME, drivers)
+function make_inputs_df(
+    site_ID;
+    FT = Float64,
```

Comment:
I'm hesitant about redefining FT here. FT should be defined once in the driver to avoid Float32/Float64 conflicts. It's not used in this function so I think we can remove it here

Reply:
I actually had to put it back after removing FT definition from the module (and putting it fluxnet_simulation), it is used in domain_setup.jl 

(I think this function make_inputs_df is not a priority, it is just used for plotting. We can think more about that later)

Reply:
Hmm, maybe we should define `FT` in experiments/ozark.jl and pass it to both `fluxnet_simulation` and `make_inputs_df`. It looks like domain isn't used in the function, so we could remove both `FT` and `domain` args